command-expansion-server/Dockerfile

```diff
@@ -1,4 +1,3 @@
 FROM node:lts-alpine
-COPY . /app
 WORKDIR /app
-CMD [ "npm", "start" ]
+CMD [ "npm", "run", "dev" ]
```

Comment:
Do we want to do `npm run dev` as the default here?

Reply:
For some reason I thought we had a different docker image for production. With that in mind, I'm going to completely redo this PR and its going to be more necessary. As it is, the docker image is not sufficient for production (it doesn't do a build internal to the docker image, who knows where its getting the build files).